test/functional/test_site/expected/bugs/index.html

```diff
@@ -20,8 +20,11 @@
     <link rel="icon" href="/test_site/favicon.png">
 </head>
 <body>
-<div id="app">
-    <div id="content-wrapper">
+    <div id="app">
+        
```

Comment:
There seems to be a number of extraneous blank lines in the output html. Is this intended?

Reply:
I have not looked into this in depth but I might suspect it is due to `prepareTemplateData` defaulting to empty string values. 

I should test using the `<%_ if %>` syntax to avoid inserting empty strings and see if there are improvements.

Reply:
**Update**

Tested with `<% if (...) %>` syntax but does not get rid of the blank lines.

However, adding either `-%>` or `_%>` to the end of the template variable insertion works. 

**Interesting finding:** 
The indentation before the template variables is only applied to the first line of the inserted html instead of all the lines within the inserted html. I would assume `page.ejs` is not smart enough to tell apart simple strings vs proper html :P